Makefile

```diff
@@ -8,20 +10,28 @@ SPEC_PATH    := spec/
 
 .PHONY: default
 default: ci
-.PHONY: bash
-bash:
-	@$(COMPOSE_DEV) $(BASH)
 
 .PHONY: ci
 ci:
 	@$(BASH_TEST) "bin/rails db:setup db:migrate ci"
 
-.PHONY: clean
-clean:
-	rm -r data || true
-	$(COMPOSE_TEST) run vets-api rm -r coverage log tmp .git || true
+.PHONEY: ci-down
+ci-down:
 	$(COMPOSE_TEST) down
 
+.PHONY: ci-clean
+ci-clean:
+	$(COMPOSE_TEST) run vets-api git clean -fdx || true
```

Comment:
TIL `git clean`, so 👍 .

Does `git clean` write to output what is being deleted? A use-case may exist where a developer is generating some file and wants to see it after a build. 

I think `git clean` is more elegant, but what we had before was more explicit.

Reply:
This will output what it removes, yes.  

FWIW, as an old user of autotools `git clean` is what I expect a `make clean` target should do, while the previous behavior was what I would have expected a `make dist` or `make proper` to do.

Reply:
So long as it's printing out what's being deleted, then 👍 . It's a better catch-all for cleaning the space.

Reply:
so keep the clean task or whack it per above comment?

Reply:
this is removed per other comment